gdal/configure.ac

```diff
@@ -5600,6 +5600,22 @@ AC_SUBST(RASDAMAN_ENABLED,$RASDAMAN_ENABLED)
 AC_SUBST(RASDAMAN_INC,    $RASDAMAN_INC)
 AC_SUBST(RASDAMAN_LIB,    $RASDAMAN_LIB)
 
+
+dnl ---------------------------------------------------------------------------
+dnl brunsli support
+dnl ---------------------------------------------------------------------------
+
+AC_ARG_WITH(brunsli,[  --with-brunsli    Include brunsli support via libbrunsli library],,)
+
+BRUNSLI_ENABLED=no
+
+if test "$with_brunsli" != "" -a "$with_brunsli" != "no" ; then
+  BRUNSLI_ENABLED="yes"
+  AC_SUBST(BRUNSLI_ENABLED, $BRUNSLI_ENABLED)
+  AC_SUBST(BRUNSLI_INCLUDE, -I$prefix/include)
+  AC_SUBST(BRUNSLI_LIB, "-lbrunslicommon -lbrunslienc -lbrunslidec -lbrotlicommon -lbrotlienc -lbrotlidec")
```

Comment:
`BRUNSLI_LIB` is super fragile:

- lib ordering is wrong: brotlicommon is a dependency of brotlienc and brotlidec, same for brunsli.

- Lib names of brunsli seem wrong: https://github.com/google/brunsli/blob/v0.1/brunsli.cmake#L50-L85

- brolti lib names are different if static (`-static` suffix)

Reply:
we'll probably not put too much effort in autoconf support for Brunsli, but I've filed https://github.com/OSGeo/gdal/issues/5068 to add support for it to CMake builds

Reply:
@SpaceIm 

>     * Lib names of brunsli seem wrong: https://github.com/google/brunsli/blob/v0.1/brunsli.cmake#L50-L85. Installation of brunsli is not neat, but CMakeLists suggests it's always shared and lib names are `brunslidec-c` and `brunslienc-c`.
> 

Those are for the C API, which is much more limited than the C++ one if I recall.

Reply:
I see, but it's worth noting that the very few package managers having this lib don't even packaged the static libs (C++), only the shared C lib.
Nevertheless, static lib names have -static suffix as you can see in CMakeLists.